proposals/graduation/cubefs-graduation-dd.md

```diff
@@ -1,7 +1,7 @@
 # CubeFS Graduation Due Diligence
 
 - Link to [CubeFS Graduation application](https://github.com/cncf/toc/pull/1140)
-
+  
```

Comment:
It seems there are unnecessary space character changes, would be helpful if you discard them